node/src/accountant/db_access_objects/failed_payable_dao.rs

```diff
@@ -578,50 +592,34 @@ mod tests {
         let now = current_unix_timestamp();
         let tx1 = FailedTxBuilder::default()
             .hash(make_tx_hash(1))
-            .reason(FailureReason::PendingTooLong)
+            .reason(PendingTooLong)
+            .timestamp(now - 3600)
             .rechecked(false)
-            .timestamp(now - 3600) // 1 hour ago
             .build();
         let tx2 = FailedTxBuilder::default()
             .hash(make_tx_hash(2))
-            .reason(FailureReason::PendingTooLong)
-            .rechecked(true) // This one is rechecked
-            .timestamp(now - 7200) // 2 hours ago
-            .build();
-        let tx3 = FailedTxBuilder::default()
-            .hash(make_tx_hash(3))
-            .reason(FailureReason::PendingTooLong)
+            .reason(NonceIssue)
             .rechecked(false)
-            .timestamp(now - 1800) // 30 minutes ago
             .build();
-        let tx4 = FailedTxBuilder::default()
+        let tx3 = FailedTxBuilder::default()
             .hash(make_tx_hash(4))
-            .reason(FailureReason::NonceIssue)
+            .reason(PendingTooLong)
             .rechecked(false)
-            .timestamp(now - 3600) // 1 hour ago
-            .build();
-        let tx5 = FailedTxBuilder::default()
-            .hash(make_tx_hash(5))
-            .reason(FailureReason::PendingTooLong)
-            .rechecked(true) // This one is rechecked
-            .timestamp(now - 3000) // 50 minutes ago
+            .timestamp(now - 3000)
             .build();
 
         subject
-            .insert_new_records(&vec![tx1.clone(), tx2, tx3.clone(), tx4, tx5])
+            .insert_new_records(&vec![tx1.clone(), tx2, tx3.clone()])
             .unwrap();
 
-        // Retrieve unchecked PendingTooLong transactions from the last hour
-        let result = subject.retrieve_txs(Some(FailureRetrieveCondition::UncheckedPendingTooLong(
-            3600,
-        )));
+        let result = subject.retrieve_txs(Some(FailureRetrieveCondition::UncheckedPendingTooLong));
```

Comment:
Actually, this should be the act of this test. Don't isolate `insert_new_records`. (It's the fake prophet :laughing: )